README.md

```diff
@@ -267,6 +268,29 @@ As long as your are running locally, use the value `dev`.
 
 3. Save the .env file.
 
+
+## Adding Packages
+
+(Pipenv)[https://pipenv.pypa.io/en/latest/] is a packaging tool for Python that solves some common problems associated with the typical development workflow using pip, virtualenv, and the requirements.txt. In addition to addressing some common issues, it consolidates and simplifies the development process to a single command line tool.
```

Comment:
Looks like the () and [] are flipped flopped here. Should be [Pipenv](https://pipenv.pypa.io/en/latest/)